Adaptors/Amqp/src/PullQueueStorage.cs

```diff
@@ -90,8 +91,10 @@ public class PullQueueStorage : QueueStorage, IPullQueueStorage
     logger_ = logger;
   }
 
-  public new ValueTask<bool> Check(HealthCheckTag tag)
-    => ValueTask.FromResult(isInitialized_);
+  public new Task<HealthCheckResult> Check(HealthCheckTag tag)
```

Comment:
What does this `new` means?

Reply:
It seems to say that you want to implement another version of this method

Reply:
https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/knowing-when-to-use-override-and-new-keywords